instrumentation/opentelemetry/init.go

```diff
@@ -434,6 +435,7 @@ func initializeMetrics(cfg *config.AgentConfig, versionInfoAttrs []attribute.Key
 	meterProvider := metric.NewMeterProvider(metric.WithReader(periodicReader), metric.WithResource(metricResources))
 	otel.SetMeterProvider(meterProvider)
 
+	metrics.InitialiseMetrics()
```

Comment:
I think `InitializeSystemMetrics` would be better as we are calling it from here